api.js

```diff
@@ -32,6 +32,18 @@ const login = async (email, password) => {
   return await axios.post(`${baseurl}/users/session`, data, getConfig(auth = false));
 };
 
+const createLeague = async (name, sp, start, end) => {
```

Comment:
Nitpicky but the variable names sp, start and end aren't descriptive.

I think you should have it as startPosition, startDate, endDate.

Also, you can parse the startPosition before calling this method to avoid the naming conflict I see on line 37.

Reply:
in